modules/viewing-kueue-alerts-for-distributed-workloads.adoc

```diff
@@ -0,0 +1,96 @@
+:_module-type: PROCEDURE
+
+[id="viewing-kueue-alerts-for-distributed-workloads_{context}"]
+= Viewing Kueue alerts for distributed workloads
+
+[role='_abstract']
+
+In {productname-short}, you can view Kueue alerts for your cluster.
+Each alert provides a link to a _runbook_.
+The runbook provides instructions on how to resolve the situation that triggered the alert.
+
+.Prerequisites
+ifdef::upstream,self-managed[]
+* You have logged in to {openshift-platform} with the `cluster-admin` role.
+endif::[]
+ifdef::cloud-service[]
+* You have logged in to OpenShift with the `cluster-admin` role.
+endif::[]
+
+ifndef::upstream[]
+* You can access a data science cluster that is configured to run distributed workloads as described in link:{rhoaidocshome}{default-format-url}/working_with_distributed_workloads/configuring-distributed-workloads_distributed-workloads[Configuring distributed workloads].
+endif::[]
+ifdef::upstream[]
+* You can access a data science cluster that is configured to run distributed workloads as described in link:{odhdocshome}/working-with-distributed-workloads/#configuring-distributed-workloads_distributed-workloads[Configuring distributed workloads].
+endif::[]
+
+ifndef::upstream[]
+* You can access a data science project that contains a workbench, and the workbench is running a default notebook image that contains the CodeFlare SDK, for example, the *Standard Data Science* notebook. 
+For information about projects and workbenches, see link:{rhoaidocshome}{default-format-url}/working_on_data_science_projects[Working on data science projects].
+endif::[]
+ifdef::upstream[]
+* You can access a data science project that contains a workbench, and the workbench is running a default notebook image that contains the CodeFlare SDK, for example, the *Standard Data Science* notebook. 
+For information about projects and workbenches, see link:{odhdocshome}/working-on-data-science-projects[Working on data science projects].
+endif::[]
+
+* You have logged in to {productname-long}.
+* Your data science project contains distributed workloads.
+
+.Procedure
+
+ifdef::upstream,self-managed[]
+. In the {openshift-platform} console, click *Observe* -> *Alerting*.
+endif::[]
+ifdef::cloud-service[]
+. In the OpenShift console, click *Observe* -> *Alerting*.
+endif::[]
+
+. Click the *Alerting rules* tab to view a list of all alerting rules. 
+
+* The *Severity* column indicates whether the alert is informational, a warning, or critical.
+* The *Alert state* column indicates whether a rule is currently firing.
```

Comment:
```suggestion
* The *Alert state* column indicates whether a rule is currently active.
```
A bit nitpicky so I don't mind if you toss this one - "firing" has a slightly negative connotation and feels a bit colloquial.

Reply:
I do see that this is used in the UI - I'm happy to set this aside if you need to reflect the UI. If you decide to go with this change, there are a few other instances later in the doc that should be updated as well.

Reply:
Thank you @eturner24. I had a similar concern but felt it was important to be consistent with the UI and with the OpenShift documentation, where the term "active" is used for the "Pending" state, and the term "firing" is used for the "Firing" state.